flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -178,7 +172,7 @@
 	 */
 	@Test
 	public void testEarlyCanceling() throws Exception {
-		Deadline deadline = new FiniteDuration(2, TimeUnit.MINUTES).fromNow();
```

Comment:
We have our own `Deadline` implementation that is pretty much a drop-in replacement for the scala version.

Reply:
thanks, changed